daffodil-test/src/test/resources/org/apache/daffodil/section14/sequence_groups/SequenceGroup.tdml

```diff
@@ -68,7 +68,7 @@
     </tdml:document>
     <tdml:infoset>
       <tdml:dfdlInfoset>
-        <complexEmptyContent></complexEmptyContent>
+        <complexEmptyContent/>
```

Comment:
Does the TDML runner view these as not equivlent? If so we should open a bug.It shouldn't matter if the TDML infoset contains ``<foo></foo>`` or ``<foo/>``.

Reply:
It does see them as equivalent, but I just wanted to use one test to make sure that it was outputting stuff correctly and had modified this test.  I can revert it back as it doesn't really make a difference.